ompi/runtime/ompi_rte.h

```diff
@@ -69,7 +69,7 @@
  *     1. ompi_rte_collective_t - an OPAL object used during RTE collective operations
  *        such as modex and barrier. It must be an opal_list_item_t and contain the
  *        following fields:
- *           a. id (ORTE type: int32_t)
+ *           a. id (pmix type: int32_t)
```

Comment:
FWIW: this probably needs to be rewritten - OMPI procs no longer interact with a separate RTE layer, so there is no mapping of types or objects any more. We don't even have RTE components as that framework has been eliminated. Much of the verbiage in this file should, therefore, be removed.

Reply:
Yeah - this area needs more scrubbing.